requirements_dev.txt

```diff
@@ -67,19 +67,19 @@ pytest-factoryboy==2.0.3  # via -r requirements_dev.in
 pytest==6.1.2             # via -r requirements_dev.in, pytest-cov, pytest-django, pytest-factoryboy
 python-dateutil==2.8.1    # via -r ./requirements.txt, django-datetime-utc, faker
 pytz==2020.4              # via -r ./requirements.txt, django
-pyzmq==19.0.2             # via locust
-requests==2.24.0          # via -r ./requirements.txt, coreapi, locust
+pyzmq==20.0.0             # via locust
```

Comment:
Prima, geen breaking changes:

https://pyzmq.readthedocs.io/en/latest/changelog.html#id1

Reply:
zmq wordt sowieso niet gebruikt in locust. Dus geen problemo.